doc/api/repl.markdown

```diff
@@ -64,6 +64,9 @@ the following values:
    returns the formatting (including coloring) to display. Defaults to
    `util.inspect`.
 
+ - `completer` - an optional function that is used for Tab autocompletion.
+   See [Readline Interface][] for an example of using this.
```

Comment:
I think you need to use https://nodejs.org/api/readline.html#readline_readline_createinterface_options